feat: add on_raw_member_* events (#643)
DisnakeDev/disnake: changelog/638.feature.rst

Add on_raw_member_remove and on_raw_member_update events.

> Along with these new events are the RawGuildMemberRemoveEvent and RawGuildMemberUpdateEvent models.

Author: c45bc96

> Add :func:`on_raw_member_remove` and :func:`on_raw_member_update` events.
> 
> Along with these new events are the :class:`RawGuildMemberRemoveEvent` and :class:`RawGuildMemberUpdateEvent` models.

Reviewer: ```suggestion
Add :func:`on_raw_member_remove` and :func:`on_raw_member_update` events, with their respective :class:`RawGuildMemberRemoveEvent` and :class:`RawGuildMemberUpdateEvent` models.
```
Multiline changelog entries will be rendered in a bit of a weird way, it's generally better to have things on one line or use bullet points for the following lines.

Author: 9e8ad04
